packages/ckeditor5-table/src/tablecolumnresize/utils/getsmallestcolumnresizer.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/tablecolumnresize/utils/getsmallestcolumnresizer
+ */
+
+import type { Editor } from 'ckeditor5/src/core.js';
+import type { ViewElement } from 'ckeditor5/src/engine.js';
+
+/**
+ * In this scenario:
+ *
+ *  +---+---+---+
+ *  | a         |
+ *  +---+---+---+
+ *  | b | c | d |
+ *  +---+---+---+
+ *  | e | f | g |
+ *  +---+---+---+
+ *
+ * When user selects column that contains `a`, `b`, `e` cells this function returns the first smallest
+ * column resize view element in that selection (`b` column).
+ *
+ * @param editor Editor instance.
+ * @returns View element of column resizer DOM node.
+ */
+export function _getSmallestColumnResizer( editor: Editor ): ViewElement | null {
```

Comment:
It's only used in the command, so it may as well be a local function defined there. Or if you think it may be useful somewhere else, we have `utils.ts` or even `TableColumnResizeUtils` plugin now.

Reply:
Moved to `TableColumnResizeUtils`.